packages/charts/src/charts/table/header-row.component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { HeaderCeil, SortMode, SortByType, Typography } from '@keen.io/ui-core';
+import { Header } from './table.styles';
+
+type Props = {
+  data: string[];
+  color: string;
+  onClick?: (res: { property: string; sort: SortMode }) => void;
+  sorting?: SortByType;
+  onResize?: (res: { property: string; width: number }) => void;
+  typography: Typography;
+};
+
+export const HeaderRow = ({
+  data,
+  color,
+  onClick,
+  sorting,
+  onResize,
+  typography,
+}: Props) => {
+  const [dragged, setDragged] = useState(false);
+
+  return (
+    <Header color={color} typography={typography}>
+      {data.map((item: any) => (
```

Comment:
Based on Props definition the `item` will be just `string` -> but in loop there is reference to `key` and `value` properties. Could You please make this type more precise.